state_machine/applications/flight/lib/mekf.py

```diff
@@ -46,31 +47,28 @@ def step(
     A = block([
         [R,              (-δt * I(3))],
         [zeros((3, 3)),  I(3)]])
-    P_p = matmul(A, matmul(P, A.tranpose())) + W
+    P_p = matmul(A, matmul(P, A.transpose())) + W
 
     # Innovation
-
-    Q = quaternion_to_left_matrix(q_p).transpose()
+    Q = quaternion_to_rotation_matrix(q_p).transpose()
     body_measurements = block([[br_mag],
                                [br_sun]])
     inertial_measurements = block([[nr_mag],
                                    [nr_sun]])
-    inertial_to_body = block([[Q,            zeros(3, 3)],
-                              [zeros(3, 3),  Q]])
+    inertial_to_body = block([[Q,              zeros((3, 3))],
+                              [zeros((3, 3)),  Q]])
     Z = body_measurements - matmul(inertial_to_body, inertial_measurements)
-    C = block([[hat(ᵇr_mag), zeros(3, 3)],
-               [hat(ᵇr_sun), zeros(3, 3)]])
-    S = C * P_p * C.transpose() + V
+    C = block([[hat(ᵇr_mag), zeros((3, 3))],
+               [hat(ᵇr_sun), zeros((3, 3))]])
+    S = matmul(C, matmul(P_p, C.transpose())) + V
```

Comment:
I feel like this line can be broken down (for clarity's sake) - not a major concern though

Reply:
I think splitting it up onto two lines would just make it more confusing honestly, I'll add a comment with the formula however.
Something like `# S = CP_pC'+V`